packages/common-scripts/src/omnilock.ts

```diff
@@ -49,6 +49,7 @@ export type OmnilockInfo = {
 export type OmnilockAuth =
   | IdentityCkb
   | IdentityEthereum
+  | IdentityEthereumDisplaying
```

Comment:
What does `Identity Ethereum Displaying` mean

Reply:
https://github.com/cryptape/omnilock/blob/cd764d7133ec4e6b192fac4b93fc0596ef5b71f6/c/ckb_identity.h#L72

I used the same name as the source from Omnilock

Reply:
Got it